parachain/primitives/router/src/inbound/mod.rs

```diff
@@ -318,3 +295,19 @@ impl<AccountId> GlobalConsensusEthereumConvertsFor<AccountId> {
 		(b"ethereum-chain", chain_id).using_encoded(blake2_256)
 	}
 }
+
+#[allow(clippy::large_enum_variant)]
+#[derive(Encode, Decode, Debug, PartialEq, Eq, Clone, TypeInfo)]
+pub enum Call {
+	/// Call create_asset on foreign assets pallet.
+	#[codec(index = 53)]
```

Comment:
I think the pallet index should stay configured in the runtime config. What happens if the ForeignAssets pallet indexes are different per runtime? I think they are the same and that is the convention, but I still think this is runtime related and shouldn't live in the snowbridge crates.

Reply:
> What happens if the ForeignAssets pallet indexes are different per runtime? I think they are the same and that is the convention.

Yeah, that's the precondition, checked that all the same in multiple runtimes(e.g. polkadot/kusama). I do not see a reason why they need to break this rule in the future so prefer to make things simple at this moment.

 But I'm open if you all think that's a problem and will make some changes accordingly.

Reply:
Maybe we can add a test to make sure they are in line?

Reply:
Ah I see one is added. Ignore this.

Reply:
I agree with Clara here, I am a bit uncomfortable with baking runtime configuration inside our pallets & primitives crate.

Secondly, I am sensitive about changing code unnecessarily at this point. Our next audit is scheduled for Feb 8, and this change will increase the scope of the audit.

I want to reserve any scope increases to accommodate for the second round of code review that the Parity bridges team is going to perform soon.

Reply:
The initial intention of this PR is for the [comment](https://github.com/paritytech/polkadot-sdk/pull/2522#discussion_r1427682795) to add the call index to some primitives which can be referenced/checked in the AH runtime tests. 

So @vgeddes do we still need this PR?  I think we already have the emulated test register token to cover this use case so maybe the refactoring here and the unit test added in AH may not be necessary at all?

Reply:
I think Slava wanted us to move those constants here: `polkadot-sdk/bridges/primitives/chain-asset-hub-rococo`. I think it is fine to move the pallet index, but not to our crates. It should still be in the polkadot-sdk, and it should be configured per testnet. I think it can actually go here too: `polkadot-sdk/cumulus/parachains/common/src/rococo.rs`

Reply:
Thanks Clara and that makes sense!  Then I think it's also better to move the [hard-coded weight](https://github.com/Snowfork/snowbridge/blob/d77201bc370ab2a4da15de701871b4a643bd5b5c/parachain/primitives/router/src/inbound/mod.rs#L191) to some common runtime config.

But if as Vincent mentioned the primary concern is to `reserve any scope increases to accommodate for the second round of code review that the Parity` then perhaps it's not the right time to do the refactoring?

So I'll stop here and let the team decide whether to continue the change or not.

Reply:
@claravanstaden Some refactoring in https://github.com/Snowfork/snowbridge/pull/1112/commits/37aa97eedcd0c3a13be72d78603a338ed1f332d0 as you suggested.